boot/jaxrs/src/main/java/org/commonjava/indy/boot/jaxrs/IndyDeployer.java

```diff
@@ -152,6 +152,7 @@ private Undertow buildAndStartUndertow( DeploymentManager dm, Integer port, Stri
     static
     {
         NO_NEED_GZIPPED_CONTENT.add( ApplicationContent.application_gzip );
+        NO_NEED_GZIPPED_CONTENT.add( ApplicationContent.application_zip );
```

Comment:
Maybe we should also add other archive content like x-tar, 7-zip or x-jar. But this time it is ok just for zip.

Reply:
Yeah, tar and jar looks quite small and mostly zip has larger size.